<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FiltrePerso</class>
 <widget class="QDialog" name="FiltrePerso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>441</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QDoubleSpinBox" name="c50"/>
    </item>
    <item row="4" column="6">
     <widget class="QDoubleSpinBox" name="c46"/>
    </item>
    <item row="4" column="5">
     <widget class="QDoubleSpinBox" name="c45"/>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="c51"/>
    </item>
    <item row="5" column="3">
     <widget class="QDoubleSpinBox" name="c53"/>
    </item>
    <item row="4" column="0">
     <widget class="QDoubleSpinBox" name="c40"/>
    </item>
    <item row="2" column="6">
     <widget class="QDoubleSpinBox" name="c26"/>
    </item>
    <item row="4" column="3">
     <widget class="QDoubleSpinBox" name="c43"/>
    </item>
    <item row="4" column="2">
     <widget class="QDoubleSpinBox" name="c42"/>
    </item>
    <item row="5" column="6">
     <widget class="QDoubleSpinBox" name="c56"/>
    </item>
    <item row="5" column="2">
     <widget class="QDoubleSpinBox" name="c52"/>
    </item>
    <item row="0" column="2">
     <widget class="QDoubleSpinBox" name="c02"/>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="c01"/>
    </item>
    <item row="0" column="5">
     <widget class="QDoubleSpinBox" name="c05"/>
    </item>
    <item row="1" column="0">
     <widget class="QDoubleSpinBox" name="c10"/>
    </item>
    <item row="2" column="0">
     <widget class="QDoubleSpinBox" name="c20"/>
    </item>
    <item row="5" column="4">
     <widget class="QDoubleSpinBox" name="c54"/>
    </item>
    <item row="5" column="5">
     <widget class="QDoubleSpinBox" name="c55"/>
    </item>
    <item row="0" column="6">
     <widget class="QDoubleSpinBox" name="c06"/>
    </item>
    <item row="1" column="2">
     <widget class="QDoubleSpinBox" name="c12"/>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="c11"/>
    </item>
    <item row="0" column="3">
     <widget class="QDoubleSpinBox" name="c03"/>
    </item>
    <item row="0" column="4">
     <widget class="QDoubleSpinBox" name="c04"/>
    </item>
    <item row="1" column="4">
     <widget class="QDoubleSpinBox" name="c14"/>
    </item>
    <item row="1" column="3">
     <widget class="QDoubleSpinBox" name="c13"/>
    </item>
    <item row="0" column="0">
     <widget class="QDoubleSpinBox" name="c00"/>
    </item>
    <item row="1" column="5">
     <widget class="QDoubleSpinBox" name="c15"/>
    </item>
    <item row="1" column="6">
     <widget class="QDoubleSpinBox" name="c16"/>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="c21"/>
    </item>
    <item row="4" column="4">
     <widget class="QDoubleSpinBox" name="c44"/>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="c41"/>
    </item>
    <item row="3" column="6">
     <widget class="QDoubleSpinBox" name="c36"/>
    </item>
    <item row="2" column="2">
     <widget class="QDoubleSpinBox" name="c22"/>
    </item>
    <item row="2" column="3">
     <widget class="QDoubleSpinBox" name="c23"/>
    </item>
    <item row="2" column="5">
     <widget class="QDoubleSpinBox" name="c25"/>
    </item>
    <item row="2" column="4">
     <widget class="QDoubleSpinBox" name="c24"/>
    </item>
    <item row="3" column="5">
     <widget class="QDoubleSpinBox" name="c35"/>
    </item>
    <item row="3" column="2">
     <widget class="QDoubleSpinBox" name="c32"/>
    </item>
    <item row="3" column="4">
     <widget class="QDoubleSpinBox" name="c34"/>
    </item>
    <item row="3" column="3">
     <widget class="QDoubleSpinBox" name="c33"/>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="c31"/>
    </item>
    <item row="3" column="0">
     <widget class="QDoubleSpinBox" name="c30"/>
    </item>
    <item row="6" column="0">
     <widget class="QDoubleSpinBox" name="c60"/>
    </item>
    <item row="6" column="1">
     <widget class="QDoubleSpinBox" name="c61"/>
    </item>
    <item row="6" column="2">
     <widget class="QDoubleSpinBox" name="c62"/>
    </item>
    <item row="6" column="3">
     <widget class="QDoubleSpinBox" name="c63"/>
    </item>
    <item row="6" column="4">
     <widget class="QDoubleSpinBox" name="c64"/>
    </item>
    <item row="6" column="5">
     <widget class="QDoubleSpinBox" name="c65"/>
    </item>
    <item row="6" column="6">
     <widget class="QDoubleSpinBox" name="c66"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>143</width>
     <height>109</height>
    </rect>
   </property>
   <property name="title">
    <string>Taille de la matrice</string>
   </property>
   <widget class="QRadioButton" name="radioButton3x3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>3x3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton5x5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>5x5</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton7x7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>7x7</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FiltrePerso</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FiltrePerso</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
